src/component/race/Race.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import octane from "./images/octane.png";
+
+function Race(props) {
+  const racePositions = 15;
```

Comment:
I like this being in a constant, but doesn't seem right to hard code.

Reply:
@jgayfer I think the reason for hard coding it right now is because there's no styled race design to have the image move inside of. Once there is something, the 'racePositions' so to speak will be set based on the size of the race box etc. Does that make sense?

Reply:
A more robust solution would be to use a percentage based style, such that we don't need to know the width of the container.